modules/azure/billing-alert-functions/main.tf

```diff
@@ -0,0 +1,171 @@
+# Code adapted from:
+# https://github.com/dzeyelid/azure-cost-alert-webhook-to-slack/blob/main/iac/terraform/modules/mediation_functions/main.tf
+
+terraform {
+  required_providers {
+    azurerm = {
+      source  = "hashicorp/azurerm"
+      version = "~> 2.96.0"
+    }
+  }
+}
+
+data "azurerm_client_config" "azurerm_provider" {
+  provider = azurerm
+}
+
+data "azurerm_subscription" "subscription" {
```

Comment:
It doesn't look like this is being used, is this needed?

Reply:
Was not needed. Taken out